style/style .css

```diff
@@ -0,0 +1,91 @@
+header {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    background-color: black;
+    height: 200px;
+    border-bottom: yellow solid 3px;
+  }
+  .logo {
+    color: yellow;
+    font-size: 70px;
+    padding: 10px;
+  }
+  nav {
+    /* justify-items: flex-end; */
+    align-self: flex-end;
+    padding: 10px;
+    display: flex;
+    justify-content: space-around;
+  }
+  nav a {
+    color: white;
+    font-size: 25px;
+    text-decoration: none;
+    padding: 10px;
+  }
+  .are {
+    text-decoration: yellow underline;
+  }
+  .taste {
+    text-decoration: line-through;
+    color: gray;
+    text-align: center;
+  }
+  main {
+    background-image: url('../images/coworking.jpg');
+    height: 700px;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+  section {
+    height: 50px;
+    background-color: lightgray;
+    font-size: 30px;
+    text-align: center;
+    padding-top: 10px;
+  }
+  img{ 
+    width: 100%;
+    height: 1000px;
+  
+  }
+  .image-container{
+    display: flex;
+    position: relative;
+    justify-content: flex-end;
+    
+    
+  }
+  .text-box{
+    
+    position: absolute;
+    background-color:  lightgrey ;
+    width: 250px;
+    height: 400px;
+    border:  2px solid black;
+    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    margin-right: 1000px;
```

Comment:
Try to use `top: 5%;` & `right: 5%;` rather than the margin, because you used `position: absolute;`
see more [CSS position](https://www.w3schools.com/cssref/pr_class_position.asp)